src/main/java/io/molr/gui/fx/widgets/MissionPane.java

```diff
@@ -168,15 +172,39 @@ private void instantiate(Map<String, Object> params) {
 
     private void configureForInstance(MissionHandle handle) {
         instanceInfo.getChildren().setAll(new Label(handle.toString()));
+        disposeButton = new FormattedButton("Dispose", "Instantiate", "Blue");
+        disposeButton.getButton().setDisable(true);
+        disposeButton.getButton().setOnAction(event -> {
+            mole.instruct(handle, MissionCommand.DISPOSE);
+        });
+        instanceInfo.getChildren().add(disposeButton.getButton());
+
+        Flux<MissionOutput> missionOutputsFlux = mole.outputsFor(handle).publishOn(fxThread());
+        missionOutputsFlux.subscribe(output->{}, error->{} , this::onOutputsComplete);
+        Flux<MissionState> missionStateFlux = mole.statesFor(handle).publishOn(fxThread());
+        missionStateFlux.subscribe(this::updateStates, error -> {}, this::onStatesComplete);
 
-        mole.statesFor(handle).publishOn(fxThread()).subscribe(this::updateStates);
         mole.outputsFor(handle).publishOn(fxThread()).subscribe(this::updateOutput);
         mole.representationsFor(handle).publishOn(fxThread()).subscribe(this::updateRepresentation);
 
         addInstanceColumns();
 
         setBottom(createOutput());
     }
+    
+    private void onOutputsComplete() {
+        /**
+         * TODO use case?
+         */
+    }
+    
+    private void onStatesComplete() {
+        
+        /*
+         * how should we detect and handle, that mission has been disposed in mission pane
```

Comment:
I am not sure if there is really a particular handling necessary .. there might just be no updates anymore ... or?
Another idea is that we include something in the mission state, saying that it is disposed .. we will see what we need....